src/lib/components/particles/waffle/waffle.stories.jsx

```diff
@@ -0,0 +1,43 @@
+import { Waffle } from '.'
+
+const meta = {
+  title: 'Particles/Waffle',
+  component: Waffle,
+}
+export default meta
+
+// this should also be the sort order, from bigger party to smaller with undeclared at the end
+const summary = {
+  lab: 350,
+  con: 200,
+  snp: 50,
+  ld: 30,
+  green: 1,
+  undeclared: 19
+}
+
+
+const partySeats = Object.keys(summary).map(party => Array(summary[party]).fill({ party })).flat()
```

Comment:
It is somewhat difficult to understand what the output of this line is. Is there a way to make this easier to read, maybe using a simpler example for the story?